pyanaconda/constants.py

```diff
@@ -201,21 +201,11 @@
 NTP_SERVER_NOK = 1
 NTP_SERVER_QUERY = 2
 
-DISPLAY_MODE_GUI = "g"
-DISPLAY_MODE_TUI = "t"
-DISPLAY_MODE_NONINTERACTIVE_TUI = "c"
+# Display modes
+DISPLAY_MODE_GUI = 0
+DISPLAY_MODE_TUI = 1
```

Comment:
Just a side note while looking at this -- maybe it's time to move to ``Enum``s in Anaconda?